pages/en/contrib/Governance.md

```diff
@@ -20,18 +20,22 @@ Each repository has one or more lead maintainers responsible for:
 *   Seeking consensus on technical decisions.
 *   Making the final decisions when consensus cannot be achieved.
 
-Maintainers have npm publishing rights for modules and the final say on releasing new versions.
+Maintainers have npm publishing rights for modules and the final say on releasing new versions.  
 
-## Lead maintainers
+Refer to `CODEOWNERS` file in each repository for the list of lead maintainers. 
 
-LoopBack lead maintainers include:
+### Full-time maintainers
 
+Project architects:
 *   [Miroslav Bajtoš](https://github.com/bajtos)
 *   [Raymond Feng](https://github.com/raymondfeng)
-*   [Ritchie Martori](https://github.com/ritch)
-*   [Simon Ho](https://github.com/superkhau)
```

Comment:
Let's add a new section "Alumni" towards the end and list people that are no longer actively working on the project there.

Reply:
One concern of adding the alumni section is that there is probably a much bigger list (it's likely we'll miss out a few?) that we try to keep track of it through the CODEOWNERS file.  
If we want to keep Ritchie and Simon on this page because they have significant contributions to LB, we can put it as "previous project architects"?